app/liquid_tags/unified_embed/tag.rb

```diff
@@ -0,0 +1,44 @@
+module UnifiedEmbed
+  # This liquid tag is present to facilitate a unified user experience
+  # for declaring that they want a URL to have "embedded" behavior.
+  #
+  # What do we mean by embedded behavior?  A more contextually rich
+  # rendering of the URL, instead of a simple "A-tag".
```

Comment:
```suggestion
  # rendering of the URL, instead of a simple "href".
```

I think "href" is more descriptive than "A-tag"

Reply:
Our fallback is to render an A-tag.

https://github.com/forem/forem/blob/261a1025ba3b442adb2ae6e306e4858d1811400e/app/liquid_tags/unified_embed/tag.rb#L37-L40